pkg/controllers/workapplier/apply.go

```diff
@@ -233,6 +240,7 @@ func (r *Reconciler) threeWayMergePatch(
 		Resource(*gvr).Namespace(manifestObj.GetNamespace()).
 		Patch(ctx, manifestObj.GetName(), patch.Type(), data, patchOpts)
 	if err != nil {
+		_ = controller.NewAPIServerError(false, err)
```

Comment:
Is this just to print out an unexpected error as the API server error is not returned. I was thinking of seeing this on the work status so the rollout can decide if it needs to keep retry or not

Reply:
Done 👌